### PR TITLE
User can remove a place from their favorites list #5 - Henry Ding

### PR DESCRIPTION
**Pull request by henryrding**
AJAX #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,17 +69,29 @@ <h4>Unit</h4>
           </div>
         </form>
       </div>
-      <div id="more-info-page" class="hidden"></div>
+      <div id="more-info-page" class="hidden">
+        <p>hello world</p>
```
Comment (shawnkost) [inline index.html]: Probably shouldn't leave "hello world" here, the text should be more descriptive of what's actually being shown

File: index.html [hunk 2]
```diff
@@ -68,17 +69,29 @@ <h4>Unit</h4>
...
+        <img src="http://www.7timer.info/bin/civil.php?lon=-117.8&lat=33.7&ac=0&lang=en&unit=british&output=internal&tzshift=0" alt="temp">
```
Comment (shawnkost) [inline index.html]: "temp" isn't descriptive of what this image actually is. Does this mean temporary, or temperature? If it's the latter, then it should say temperature.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good! Just a couple changes requested here

File: index.html (hunk 2 above)
Comment (henryrding) [reply index.html]: It is supposed to mean temporary, it's a placeholder for an image I will get from the API with coordinates that match the place.

File: index.html (hunk 1 above)
Comment (henryrding) [reply index.html]: The p element was leftover from debugging getting the more-info-page to show when making sure view swapping was working.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**